tests/auto_indent_test.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+
+import filecmp
+import os
+import sys
+from shutil import copyfile
+
+import pytest
+
+from pre_commit_hooks.auto_indent import main
+from pre_commit_hooks.auto_indent import RETURN_CODE
+from testing.util import get_resource_path
+
+
+def _has_ast_arg_lineno_support():
+    """Checks if AST has all the info necessary to auto indent.
+
+    Some Python implementations removed line numbers from AST arg objects and
+    as a result we can't automatically format function calls.
+    """
+    if sys.version_info.major == 3 and sys.version_info.minor == 3:
+        return False
+    elif sys.version_info.major == 3 and sys.version_info.minor == 2:
+        return False
+    else:
+        return True
```

Comment:
(though you're going to remove this)

aka
```python
return sys.version_info > (2, 7) and sys.version_info < (3, 4)`
```

though ideally this would be feature detection instead:

```python
return hasattr(ast.parse(...).body[0]...
```